worker.go

```diff
@@ -590,8 +592,24 @@ func updateIngress(kubeAdapter *kubernetes.Adapter, lb *loadBalancer, problems *
 	if lb.clusterLocal {
 		dnsName = kubernetes.DefaultClusterLocalDomain
 	} else {
-		// only update ingress if the stack exists and is in a complete state.
-		if lb.stack == nil || !lb.stack.IsComplete() {
+		// only update ingress if the CF stack is in a completed state and the ELB
+		// is in the active state.
+		if lb.stack == nil {
+			log.Infof("CF stack is nil, skipping ingress update")
+			return
+		}
+		if !lb.stack.IsComplete() {
+			log.Infof(
+				"CF stack %q is not in a completed state, skipping ingress update",
+				lb.stack.Name,
+			)
+			return
+		}
+		if lb.state == nil || lb.state.Code != elbv2Types.LoadBalancerStateEnumActive {
+			log.Infof(
+				"the load balancer of CF stack %q is not in active state, skipping ingress update",
```

Comment:
Log messages should start from capital letter.
Lets log the state and maybe loadbalancer ARN as well?

Reply:
> Log messages should start from capital letter.

Interesintg! If I'm not mistaken in most of the common go libraries, they don't start logs with a capital letter.  I thought in go work logs should always start with a simple letter.

Reply:
resolving this as the suggested changes were applied